css/index.css

```diff
@@ -111,6 +132,38 @@ p {
     align-items: center;
     justify-content: center;
 }
```

Comment:
Good work using id's for each box.  These are only going to be used one time on this page.